docs/kola.md

```diff
@@ -53,6 +53,25 @@ and can also be used with glob patterns:
 
 `kola --denylist-test linux.nfs* --denylist-test crio.* run`
 
+Tests specified in `src/config/kola-denylist.yaml` will also be skipped regardless of whether the switch `--denylist-test` was provided.
```

Comment:
Break up this long line into two lines to match what's done in the rest of the file.

```suggestion
Tests specified in `src/config/kola-denylist.yaml` will also be skipped
regardless of whether the switch `--denylist-test` was provided.
```